### DISCUSSION TITLE
Progressive Web Apps

### DISCUSSION THREAD

**Discussion by JuliaAgostino** [mayrinck/FOSSonMicrosoftStore#26] [General]
I know this repository is for free open-source software, but could we consider also uploading PWAs too?

**Comment by mayrinck** [comment]
Well, the process to do that is a bit more complicated than uploading win32 apps... We need a lot of info and some access to some core files of the application to set up everything correctly so... Maybe we can't...

**Reply by JuliaAgostino** [reply]
Really? All you have to do is package it in [PWABuilder](https://www.pwabuilder.com/), right?

**Reply by mayrinck** [reply]
not exactly... Some PWAs I can use that process, others I can't... There are some data from them that I cannot retrieve, so I won't be able to add every PWA person's ask... Maybe we can try this in the near future.

**Reply by JuliaAgostino** [reply]
Can we submit PWAs that work with PWABuilder?

**Reply by mayrinck** [reply]
Well... Maybe we should create another repo for that, then?

**Reply by JuliaAgostino** [reply]
> Well... Maybe we should create another repo for that, then?

Yeah, do it! 🤩🚀
